AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -678,6 +684,20 @@ open class DeckPicker :
         }
     }
 
+    private fun migrationSuccessDialog() {
+        val builder = AlertDialog.Builder(this)
+        builder.setTitle(resources.getString(R.string.migration_successful_message))
+        builder.setMessage(resources.getString(R.string.migration_completed))
+        builder.setPositiveButton(R.string.dialog_ok) { dialogInterface, _ ->
+            dialogInterface.dismiss()
+        }
+        val dialog = builder.create()
+        dialog.show()
```

Comment:
Optional nitpick, but I prefer this syntax - feels cleaner.

Uses `show` and `positiveButton` to reduce complexity

```suggestion
AlertDialog.Builder(this).show {
            setTitle(resources.getString(R.string.migration_successful_message))
            setMessage(resources.getString(R.string.migration_completed))
            positiveButton(R.string.dialog_ok) {
                it.dismiss()
            }
        }
```